logging/Notes_on_using_TLS.md

```diff
@@ -2,7 +2,21 @@
 
 ## Overview
 
-TLS is required for communication between the Elasticsearch nodes and for accessing Elasticsearch from other components. It is also a best practice to use TLS (by using HTTPS) when accessing Kibana from a browser.
+TLS enablement for SAS Viya logging is divided into two parts:
+
+- **TLS for in-cluster communications**, which is between logging components within
+the cluster (between the Elasticsearch nodes and between Elasticsearch and other logging components). TLS must be enabled on these connections.
+
+- **TLS for communications into the cluster**, which is between a user's 
+browser and Kibana. TLS is optional on these connections, although enabling
+TLS is a best practice.
```

Comment:
Technically, enabling TLS here isn't limited to the browser->Kibana use-case.  This also covers the case of requiring TLS between ingress and Kibana (if ingress is configured).  There is also a use-case for connecting to the Elasticsearch API which requires its own TLS cert.